recipes/wntr/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "wntr" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/michaelbynum/WNTR/archive/0b30063fdace5d6149d22b883c03def3b3b42e4a.zip
+  sha256: f70bbedb1e0208fc74a53f4e50b5c73a6b758f8cffc29e15f840caf30f2632bf
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

Comment:
I tend not to use the py selector because sometimes its type has changed from int to str or vice versa.  But this seems to work so I think we can merge.  Just not that `[py2k]` is equivalent and more stable.